subprojects/logging/src/main/java/org/gradle/internal/deprecation/DeprecationMessageBuilder.java

```diff
@@ -53,6 +53,26 @@ public DeprecationMessageBuilder guidedBy(String documentationId, String section
         return this;
     }
 
+    public DeprecationMessageBuilder guidedBy(String documentationId) {
```

Comment:
This is the odd one out.  I would just follow the pattern you have for the others unless you can think of a better name (`withUserManual(...)`)